modules/account-lib/src/coin/stx/transaction.ts

```diff
@@ -157,22 +189,24 @@ export class Transaction extends BaseTransaction {
    */
   loadInputsAndOutputs(): void {
     const txJson = this.toJson();
-    if (txJson.payload.to && txJson.payload.amount) {
-      this._outputs = [
-        {
-          address: txJson.payload.to,
-          value: txJson.payload.amount,
-          coin: this._coinConfig.name,
-        },
-      ];
-
-      this._inputs = [
-        {
-          address: txJson.from,
-          value: txJson.payload.amount,
-          coin: this._coinConfig.name,
-        },
-      ];
+    if (txJson.payload.payloadType === PayloadType.TokenTransfer) {
```

Comment:
Are there no inputs/outputs for contract calls?

Reply:
Not that I can think of. Please suggest if you think there should be inputs/outputs/ for contract call.